cosmwasm/packages/sgx-vm/src/enclave.rs

```diff
@@ -101,14 +139,15 @@ impl EnclaveRuntimeConfig {
 }
 
 pub fn configure_enclave(config: EnclaveRuntimeConfig) -> SgxResult<()> {
-    let mut configured = SGX_ENCLAVE_CONFIGURED.lock().unwrap();
+    let mut configured = SGX_ENCLAVE_CONFIGURED.lock();
```

Comment:
Seems kind of strange to have this sort of pattern around a config, but I'm assuming this isn't part of the PR so w/e

Reply:
I did this in case this was somehow called from multiple threads. it should only be called once in the node init though so I'm not sure why i thought that was necessary.